tools/marine_omics/sanntis.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="sanntis_marine" name="Sanntis biosynthetic gene clusters" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
+    <description>in genomic and metagenomic data</description>
+    <macros>
+        <token name="@TOOL_VERSION@">0.1.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <edam_topics>
+        <edam_topic>topic_3387</edam_topic>
+    </edam_topics>
+    <requirements>
+        <requirement type="package" version="0.9.3.5">sanntis</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        sanntis --ip-file "$input_interpro" --outfile "output_sanntis.gff" "$input_genbank"
+    ]]></command>
+    <inputs>
+        <param name="input_interpro" type="data" format="tsv" label="Input a .tsv file coming  from InterProScan" help="Before using this tool you need to retrieve the right data by using the InterProScan tool"/>
```

Comment:
```suggestion
        <param name="input_interpro" type="data" format="tabular" label="Input the TSV file from InterProScan" help="Before using this tool you need to retrieve the right data by using the InterProScan tool"/>
```

If you check the output in Galaxy Interproscan it is a `tabular` format.

Reply:
Oh thanks !